jobs/involuntary-dissolutions/tests/unit/test_involuntary_dissolutions.py

```diff
@@ -275,27 +301,38 @@ def test_stage_2_process_update_business(app, session, test_name, status, step):
     assert batch_processing.status == status
     assert batch_processing.step == step
 
-    if test_name == 'MOVE_2_STAGE_2':
+    if test_name == 'MOVE_2_STAGE_2_FAILED':
+        assert batch_processing.notes == 'stage 1 email or letter has not been sent'
```

Comment:
It will be better to add assert for `batch_processing.status`. And same for the `test_stage_3_process()`.

Reply:
Oh, I was thinking I didn't want to duplicate lines. In here it's on line 301 and in `test_stage_3_process()` it's on line 374.

Reply:
Oh, my fault. Resolved!